.pre-commit-config.yaml

```diff
@@ -146,6 +146,13 @@ repos:
           - --fuzzy-match-generates-todo
         files: >
           \.cfg$|\.conf$|\.ini$|\.ldif$|\.properties$|\.readthedocs$|\.service$|\.tf$|Dockerfile.*$
+  - repo: https://github.com/psf/black
+    rev: 22.12.0
+    hooks:
+      - id: black
+        name: Run black (python formatter)
+        args: [--config=./pyproject.toml]
```

Comment:
Nit: this is the default and shouldn't be needed

Reply:
Yeah we could remove, but also I think having it explicit here might make it easier for anyone in the future looking where the configuration is. There are couple of ways it can be specified and `./pyproject.toml` configuration is not yet as popular as it should be so that gives a hint that wa are using it.